imap_processing/cdf/global_attrs.py

```diff
@@ -30,24 +33,63 @@ class GlobalConstantAttrs:
     ----------
     GLOBAL_BASE:
         Global file attributes, including project, source_name, discipline, PI_name,
-        PI_affiliation, instrument_type, and mission_group. This should be the same
+        PI_affiliation, and mission_group. This should be the same
         for all instruments.
 
     """
 
-    GLOBAL_BASE: ClassVar[dict] = {
+    GLOBAL_BASE: Final[dict] = {
         "Project": "STP>Solar-Terrestrial Physics",
         "Source_name": "IMAP>Interstellar Mapping and Acceleration Probe",
         "Discipline": "Solar Physics>Heliospheric Physics",
-        "PI_name": "Dr. David J. McComas",
-        "PI_affiliation": [
-            "Princeton Plasma Physics Laboratory",
-            "100 Stellarator Road, Princeton, NJ 08540",
-        ],
-        "Instrument_type": "Particles (space)",
+        # TODO: CDF docs say this value should be "IMAP"
         "Mission_group": "IMAP>Interstellar Mapping and Acceleration Probe",
     }
 
+    pi_name: tuple[str] = ("Dr. David J. McComas",)
+    pi_affiliation: tuple[str, str] = (
+        "Princeton Plasma Physics Laboratory",
+        "100 Stellarator Road, Princeton, NJ 08540",
+    )
```

Comment:
It looks like PI_NAME should only be one entry according to the CDF docs. Should this be replaced with the instrument PI instead of appended to with both the mission PI and instrument PI?

@bryan-harter would be good for you to review these CDF changes to see if there is a standard way of putting this into the files that SPDF approves of.

Reply:
Yeah, I wasn't sure if it should include the instrument PI or not. The example files I had from MAG did.

Reply:
Thanks for sending the examples to Bryan and following up with him. It seems like it is acceptable to do it this way and there is not necessarily a standard. We can update this at a later time if need be as well.

Reply:
Eric Christian pointed out to me that there's apparently no such thing as an "Instrument PI" on IMAP, they are "Instrument Leads".  So PI name would probably just be reserved for Dave.  I'll get confirmation later if we need another global attribute for Instrument Leads

Reply:
Sounds good. I'll remove this change from the PR and leave it as just Dave.